Address/index.ts

```diff
@@ -0,0 +1,17 @@
+import { isly } from "isly"
+import { Alpha2 } from "../CountryCode"
+import { GB as AddressGB } from "./GB"
+import { Generic as AddressGeneric } from "./Generic"
+import { SE as AddressSE } from "./SE"
+
+export type Address = AddressGeneric | AddressGB | AddressSE
+
+export namespace Address {
+	export type Generic = AddressGeneric
```

Comment:
Use the newly discovered export import syntax

Reply:
Basically replace
```ts
export type Generic = AddressGeneric
export const Generic = AddressGeneric
```
with
```ts
export import Generic = AddressGeneric
```